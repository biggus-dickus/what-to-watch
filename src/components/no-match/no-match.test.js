import * as React from 'react';
import {StaticRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';

import NoMatch from './no-match';

const tree = renderer.create(<Router><NoMatch /></Router>).toJSON();

it(`<NoMatch /> should render correctly`, () => expect(tree).toMatchSnapshot());
