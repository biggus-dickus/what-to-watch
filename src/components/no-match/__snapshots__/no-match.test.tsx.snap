// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> should render correctly 1`] = `
<section
  className="error-404"
>
  <h1>
    404, page not found
  </h1>
  <p>
    This is not the page you are looking for.
  </p>
  <p>
    Please go back or try going to the 
    <a
      href="/"
      onClick={[Function]}
    >
      home page
    </a>
    .
  </p>
</section>
`;
